server/src/main/java/org/elasticsearch/index/mapper/flattened/FlattenedFieldMapper.java

```diff
@@ -261,8 +263,7 @@ public BytesRef indexedValueForSearch(Object value) {
 
         @Override
         public ValueFetcher valueFetcher(SearchExecutionContext context, String format) {
```

Comment:
I just realized we should throw an error if the format is non-null, as we do for most other field types. (This is unfortunately very manual right now and easy to forget...)